vehicle_safety:
  CTE_THR : 5 # cte threshold in meters
  CTE_THR_AT_CURVE : 8 # CTE Threshold at curve
  HEAD_THR : 10 # Heading Threshold in degrees(+/-)
  GPS_FIX_THR : 10.0 #GPS FIX Threshold time in seconds
  BATT_TH : 15 # Battery Threshold in %. Below this value vehicle won't move
  WHITE_LIST : ["vehicle_safety_cpp_diagnostics: GPS","vehicle_safety_cpp_diagnostics: TrackingController","vehicle_safety_cpp_diagnostics: Emergency","vehicle_safety_cpp_diagnostics: GeoFence","vehicle_safety_cpp_diagnostics: CAN_Batt_Diagnostics"] # vehicle will stop, if these errors came up
  BLACK_LIST : ["vehicle_safety_cpp_diagnostics: Heading"] # vehicle will ignore and continue running, if the errors coming from these list.
  use_geo_fence : true
  use_inner_geo_fence : false
  geo_fence_coordinates: [
    [13.585564444929537, 79.96123159196975],
    [13.584881054339391, 79.9613978024903],
    [13.58421152386115, 79.96311317207108],
    [13.585489590870313, 79.96317221210154],
    [13.585564444929537, 79.96123159196975],
]
  # no_go_zone_coordinates: [
  #   [
  #     [13.585564444929537, 79.96123159196975],
  #     [13.584881054339391, 79.9613978024903],
  #     [13.58421152386115, 79.96311317207108],
  #     [13.585489590870313, 79.96317221210154],
  #     [13.585564444929537, 79.96123159196975],
  #   ]
  # ]
  
# [
#     [13.585273,79.9619002],
#     [13.5848689,79.9619216],
#     [13.584775,79.9628121],
#     [13.5852287,79.962788],
#     [13.585273,79.9619002],
# ]
  no_go_zone_coordinates: [
  [
    [13.5851234,79.9621254],
    [13.5849905,79.9621254],
    [13.5849853,79.9623722],
    [13.5851678,79.9624178],
    [13.5851234,79.9621254],
  ],
  [
    [13.5850687,79.9624822],
    [13.5849462,79.9624714],
    [13.5849644,79.9627638],
    [13.5851,79.9626511],
    [13.5850687,79.9624822],
  ]
]

 #mango farm fence coordinates
  # Footer
